### DISCUSSION TITLE
Fehlerhafte Initial Kalibrierung

### DISCUSSION THREAD

**Discussion by Thomas-Wolff** [NightscoutFoundation/xDrip#2097] [Q&A] [answered]
After the installation of the build of 2 May 2022 Bulid: 10d07c1 an initial calibration was requested in xDrip although 'OOP2 algorithm calibation' was set to false.

Calibrated blood levels 148 mg/dl and 151 mg/dl to sensor 135 mg/dl. The next sensor value read after calibration was 233 mg/dl.
There may be something wrong with the calibration, please verify.

At the moment updated back to the last construction from 1 May 2022.

Attached are the screenshots of the calibration table and the corresponding data table.

![Bild1](https://user-images.githubusercontent.com/59526802/166311918-47dc0b1c-c0c0-4f97-873f-c784e774ec5c.jpg)

![Bild2](https://user-images.githubusercontent.com/59526802/166311953-c2b716eb-ae6f-4da5-b1da-c90d4417e2b8.jpg)

Many greetings Thomas

**Comment by Navid200** [comment, marked as answer]
@jamorham @tzachi-dar

**Reply by jamorham** [reply]
@Thomas-Wolff Can you look in the OOP2 algorithm calibration setting and describe which of the 3 options it is currently set to (this feature changed in this release) thanks.

**Reply by Thomas-Wolff** [reply]
Haven't really changed anything else, except for the way the Bulid OOP2 was published without calibration I set the OOP2 algorithm calibration option to false.

Here is the corresponding screenshot:

![Screenshot_20220503-110609_xDrip+ 1](https://user-images.githubusercontent.com/59526802/166429085-d85a178f-438f-4a8b-8a89-d18c5794e1a8.jpg)

Thanks for your initiative Thomas

**Reply by jamorham** [reply]
Ok the screen shot is not from May 2nd version. Are you able to reinstall that one to see how the option shows there? In May 2nd there are now 3 options but it should default to what you had before. There are two places this bug could come from and your answer would let me know which place it could be. Thanks.

**Reply by Thomas-Wolff** [reply]
Yes, that is correct.
Yesterday I updated again to the construction of 01 May.

Please be patient, I am busy elsewhere at the moment. As soon as my time permits, I will update the construction from 02 May again and get back to you.

**Reply by jamorham** [reply]
Whenever you have time. I am investigating and checking here also.

**Reply by Thomas-Wolff** [reply]
Hello, I have now installed the construction from 2 May once again.

In the attached screenshots you can see what happened afterwards in xDrip. 
I have refrained from calibrating here.

![Bild1](https://user-images.githubusercontent.com/59526802/166440245-2519b1d1-a7fb-4f61-8c10-235d413dede5.jpg)

![Bild2](https://user-images.githubusercontent.com/59526802/166440263-3dfd82a9-c7bf-44d8-85f2-838b7a8acb76.jpg)

![Bild3](https://user-images.githubusercontent.com/59526802/166440299-1392afe1-3f90-4f98-9167-ce106a6af0e3.jpg)

Greetings Thomas

**Reply by jamorham** [reply]
Please try the May 3rd nightly. This should resolve the problem but because settings may already be set you should go in to the OOP calibration setting and select the one you want (eg no calibration)

**Reply by Thomas-Wolff** [reply]
I have installed the construction from 3 May, unfortunately no improvement is apparent.

Attached are the corresponding screens:

![Bild1](https://user-images.githubusercontent.com/59526802/166453718-99a67663-ed29-41fa-9a96-e5cad2d11ffa.jpg)

![Bild2](https://user-images.githubusercontent.com/59526802/166453741-ad6557bb-d804-49d6-a892-2ed6fc23226b.jpg)

![Bild3](https://user-images.githubusercontent.com/59526802/166453767-a345d134-ce3c-4898-a7d5-353d7d8cb52f.jpg)

![Bild4](https://user-images.githubusercontent.com/59526802/166453807-4b3c4082-a7a0-4293-9823-eb2b85ca0676.jpg)

If I can still help in any way, please let me know.

**Reply by jamorham** [reply]
Please tap on where you see "OOP2 algorithm calibration" and select no-calibration to restore the original behavior.

**Reply by Thomas-Wolff** [reply]
Many thanks for your support, it worked great.
I will test the calibration again tomorrow.

![Screenshot_20220503-225931_xDrip+ 1](https://user-images.githubusercontent.com/59526802/166568527-f3acadf8-d2de-4283-a071-6ea925b40b28.jpg)


Have a nice evening Thomas

**Comment by Thomas-Wolff** [comment]
Hello, I have to ask again.

Unfortunately, I have problems calibrating the FSL2 sensor, switched to calibration in the system and then carried out a double calibration. Blood 95 mg/dl and 102 mg/dl sensor without calibration was at 90 mg/dl.

The first subsequent value after calibration that was recorded by the calibrated sensor was now 149 mg/dl. This is very implausible for me and I trust it.

Enclosed is the corresponding screen:

![Screenshot_20220504-162644_xDrip+ 1](https://user-images.githubusercontent.com/59526802/166707562-636fa6a0-9806-4961-b0c8-332670841b12.jpg)

Greetings Thomas

**Reply by jamorham** [reply]
Can you describe which calibration method you are using and post a screenshot of your calibration graph thanks

**Reply by Thomas-Wolff** [reply]
I have performed a double initial calibration.

Blood was measured with the Contour Next One, one measurement on each fingertip of the left and right hand.
I washed my hands thoroughly and wiped off the first drop after embroidery to avoid getting too much tissue fluid in the drop.

Unfortunately, I can no longer post the calibration graphic, as I have already deleted it.

I offer to do another calibration in the same way and then post it with the corresponding results.

**Reply by jamorham** [reply]
If you are having problems with the calibration and inaccurate readings then select the no-calibration mode which might be better.

**Reply by Thomas-Wolff** [reply]
Yes, I agree, at the moment it is better to work without calibration.

I have tried to reproduce everything again. I have changed the system again and calibrated it.

Attached are the results and the screens:

1. calibration graphic directly after calibration before the first calibrated measurement value was received.

![Bild1](https://user-images.githubusercontent.com/59526802/166741132-d016cbb3-4adc-4354-927f-e01bdaaaf784.jpg)

2. calibration graph after receiving the first measured value after calibration.

![Bild2](https://user-images.githubusercontent.com/59526802/166741208-7db9fac2-8ec5-4af8-a88d-5f6dfe63c589.jpg)

3. Calibration table.

![Bild3](https://user-images.githubusercontent.com/59526802/166741276-05ef3712-521e-44e1-82b4-10cda30f62f7.jpg)

4. data table.

![Screenshot_20220504-185631_xDrip+ 1](https://user-images.githubusercontent.com/59526802/166741600-8cc6ad2d-99a6-4d40-af4e-e4afa3450b74.jpg)

5: Main screen.

![Screenshot_20220504-175335_xDrip+ 1](https://user-images.githubusercontent.com/59526802/166741661-319af1d7-feac-4c62-a2ab-20e8d33bd5fc.jpg)

Hope it helps a bit, have a nice evening, greetings Thomas
